lib/cadet/assessments/answer_types/programming_answer.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Cadet.Assessments.AnswerTypes.ProgrammingAnswer do
+  @moduledoc """
+  The ProgrammingQuestion entity represents a Programming question.
+  """
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  embedded_schema do
+    field(:solution_code, :string)
```

Comment:
I think just naming it `:code` is better, since this is contained inside an **answer** entity already

Reply:
@sreycodes bump

Reply:
Done